include/swift/AST/Attr.h

```diff
@@ -1661,6 +1661,45 @@ class OriginallyDefinedInAttr: public DeclAttribute {
   }
 };
 
+/// The `@package` attribute implies its following imports has
+/// external package dependencies.
+/// 
+/// Syntax definition is passed in on frontend calls.
+class PackageAttr final: public DeclAttribute,
+                         public TrailingCallArguments<PackageAttr> {
+  /// The package declaration string.
+  const StringRef PackageDeclaration;
+
+  unsigned NumArgLabels : 16;
+  Expr *Arg;
+
+public:
+  PackageAttr(SourceLoc AtLoc, SourceRange Range,
```

Comment:
Since this has a body, it should be moved into the `.cpp` file.

Reply:
AFAIK simple functions and the main initializer stay in `Attr.h`?